src/vellum/workflows/nodes/displayable/subworkflow_deployment_node/node.py

```diff
@@ -231,9 +231,18 @@ def run(self) -> Iterator[BaseOutput]:
                 message="Expected deployment name to be provided for subworkflow execution.",
             )
 
-        resolved_workflow = self._context.resolve_workflow_deployment(
-            deployment_name=deployment_name, release_tag=self.release_tag, state=self.state
-        )
+        try:
+            resolved_workflow = self._context.resolve_workflow_deployment(
+                deployment_name=deployment_name, release_tag=self.release_tag, state=self.state
+            )
+        except WorkflowInitializationException as e:
+            hashed_module = e.definition.__module__
+            raise NodeException(
+                message=e.message,
+                code=e.code,
+                raw_data={"hashed_module": hashed_module},
+            ) from e
```

Comment:
actually, wrap this around the first event `next()`d from the stream inside of `_run_resolved_workflow`